index.js

```diff
@@ -1417,6 +1417,22 @@
     return maybe.isNothing ? Left(x) : Right(maybe.value);
   });
 
+  //. ### Function
+
+  //# flip :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. Takes a binary function and two values of any type and returns
+  //. the result of applying those values to that function with its
+  //. argument order reversed.
```

Comment:
Functions are applied to values (not the other way around). It would be nice to tweak the wording to reflect this.

Also, we should mention that `flip` may also be applied to Ramda-style curried functions with arity greater than two.